### DISCUSSION TITLE
Is there a way to set the week to start on Sunday?

### DISCUSSION THREAD

**Discussion by nrenfree** [ivan-lednev/obsidian-day-planner#470] [Q&A]
Is there a way to set up my Day Planner settings, particularly for the Week Planner view, to start on Sunday instead of Monday? All my other plugins (including the Calendar plugin that I use alongside the Day Planner plugin) and calendars are set up to start on Sunday, and I'd love to keep things consistent (mostly to avoid confusing myself).

**Comment by ivan-liadniou-epam** [comment]
This feature request is in progress: #231
